src/index.html

```diff
@@ -1,13 +1,490 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="uk">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>NAMU</title>
+    <link
+      rel="shortcut icon"
+      href="./images/logo.svg"
+      type="image/x-icon"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@300;400;500;600;700&family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page">
+    <div class="page__body">
+      <header class="header" id="header">
+        <div class="header__content">
+          <div class="header__top">
```

Comment:
https://github.com/mate-academy/Museum/assets/106310796/900901f2-c570-43d2-b178-f1df8b13ad25

Reply:
changes should be smooth